hr_timesheet_default_analytic_account/models/hr_timesheet_sheet.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Coop IT Easy SCRLfs
+#   - Vincent Van Rossem <vincent@coopiteasy.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from datetime import datetime, timedelta
+
+from openerp import api, fields, models, _
+
+
+class HrTimesheetSheet(models.Model):
+    _inherit = "hr_timesheet_sheet.sheet"
+
+    def get_number_days_between_dates(self, date_from, date_to):
+        """
+        Returns number of days between two dates
+        @param date_from: string object
+        @param date_to: string object
+        @return: integer
+        """
+        datetime_from = datetime.strptime(date_from, "%Y-%m-%d")
+        datetime_to = datetime.strptime(date_to, "%Y-%m-%d")
+        difference = datetime_to - datetime_from
+        # return result and add a day
+        return difference.days + 1
+
+    def _prepare_analytic_line(self, date, account, sheet_id, user_id):
+        return {
+            "account_id": account.id,
+            "amount": 0,
```

Comment:
AFAIK amount are float but it should not cause any problem

Reply:
I now use `0.0`